 configure.ac |    6 ++++++
 1 file changed, 6 insertions(+)

New commits:
commit 3c64d866379c3ae78b95b68fffe62685037da707
Author: Bjoern Michaelsen <bjoern.michaelsen@canonical.com>
Date:   Mon Dec 10 19:06:48 2012 +0100

    no need for wget/curl when not fetching anything
    
    Change-Id: I097675450a0eed87eccf90356c81657485609d13

diff --git a/configure.ac b/configure.ac
index caac659..2b12bca 100644
--- a/configure.ac
+++ b/configure.ac
@@ -11753,6 +11753,10 @@ AC_SUBST(SCPDEFS)
 #
 WGET=
 MD5SUM=
+CURL=
+
+if test "$enable_fetch_external" != "no"; then
+
 CURL=`which curl 2>/dev/null`
 
 for i in wget /usr/bin/wget /usr/local/bin/wget /usr/sfw/bin/wget /opt/sfw/bin/wget /opt/local/bin/wget; do
@@ -11794,6 +11798,8 @@ if test -z "$MD5SUM"; then
     add_warning "no md5sum: found!"
 fi
 
+fi
+
 AC_SUBST(WGET)
 AC_SUBST(CURL)
 AC_SUBST(MD5SUM)
